src/commands/qr/index/processor.ts

```diff
@@ -184,10 +185,10 @@ async function compose(user: User, page: number) {
         rowAfterFormatter: (f, i) => {
           const d = data[i]
           const emoji = mapEmoji(d)
-
+          const url = d.category === "profile" ? HOMEPAGE_URL : buildQueryUrl(d)
```

Comment:
it's good if we can have enum for category

Reply:
noted, will try it later, tks ser